src/main/eo/org/eolang/math/number.eo

```diff
@@ -300,9 +310,46 @@
   #  You need to code function for coefficient of nth term.
   #  Accuracy can be decreased because of stack limitation.
 
-  # @todo #56:90 min. Implement exp using this taylor object.
-  #  You need to code a function for coefficient of nth term.
-  #  Accuracy can be decreased because of stack limitation.
+  [] > exp
+    switch > @
+      *
+        is-nan
+        nan
+      *
+        is-positive-infinity
+        number
+          positive-infinity
+      *
+        is-negative-infinity
+        0.0
+      *
+        TRUE
+        times.
+          exp-x0
+          exp-x1
+    as-float > value!
+    0.69314718055994530941 > ln-2!
+    as-int. > m!
+      number
+        div.
+          value
+          number.exp.ln-2
+    15000000.0 > count!
```

Comment:
@levBagryansky what is the origin of this magical number? Why don't you use 1e6 or 1e9?

Reply:
@includealex This number provides the good accuracy.